api/src/lists/interceptors/remove-list-fields.interceptor.ts

```diff
@@ -3,7 +3,7 @@ import { NestInterceptor, ExecutionContext, CallHandler } from '@nestjs/common';
 import { Observable } from 'rxjs';
 import { map } from 'rxjs/operators';
 import { plainToInstance } from 'class-transformer';
-import { ListDto } from '../dtos/list.dto';
+import { ListDto } from '../dtos/lists.dto';
```

Comment:
To be honest, I don't think we should rename anything to its plural form. All these entities represent 1 item of a class, not multiple.